FujiwaraChoki/MoneyPrinterV2
# Gracefully handles empty/non-integer input at menu. Fixes #70

This PR fixes a crash occurring when users enter non-integer or empty inputs during menu option selection. It introduces input validation within a loop, ensuring users provide valid integers corresponding to menu options. Related issue: #70  

Changes Made:
- Modified the input handling in the main() function to include a loop that checks for and validates integer input.
- Added error messaging to inform the user of invalid input and prompt for a retry.

How to Test:
1. Run the program and reach the main menu.
2. Press enter without typing anything or input a non-integer value.
3. Verify that the program now prompts for input again instead of crashing.
